pyartcd/pyartcd/locks.py

```diff
@@ -143,6 +148,36 @@ async def get_locks(self, version: str = None):
         return await redis.get_keys(pattern)
 
 
+async def enqueue_for_lock(coro: coroutine, lock: Lock, lock_name: str, lock_id: str,
+                           ocp_version: str, version_queue_name):
+    lock_manager = LockManager.from_lock(lock)
+    return await _enqueue_for_lock(coro, lock_manager, lock, lock_name, lock_id, ocp_version, version_queue_name)
+
+
+@tenacity.retry(
+    wait=tenacity.wait_fixed(600),  # wait for 10 minutes between retries
+    stop=tenacity.stop_after_attempt(600 * 24),  # wait for 24 hours
```

Comment:
Is 24 hours long enough to account for times when all versions want to do mass rebuilds? What happens if the timeout is exceeded?

Reply:
tenacity tryagain exception is raised
I think it's good, but we can tweak it in future